lib/gitlab/markdown_helper.rb

```diff
@@ -22,6 +22,15 @@ def gitlab_markdown?(filename)
       filename.downcase.end_with?(*%w(.mdown .md .markdown))
     end
 
+    # Public: Determines if the given filename has AsciiDoc extension.
+    #
+    # filename - Filename string to check
+    #
+    # Returns boolean
+    def asciidoc?(filename)
+      filename.downcase.end_with?(*%w(.adoc .asc .asciidoc))
```

Comment:
I think `.asc` should be removed. It should either be `.adoc .asciidoc` or `.adoc .ad .asciidoc`.